docs/content/configuration/auth.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: doc_page
+---
+
+# Authentication and Authorization
+
+|Property|Type|Description|Default|Required|
+|--------|-----------|-------|
+|`druid.auth.enabled`|boolean|Determines if authentication and authorization checks will be performed on requests.|false|no|
+|`druid.auth.authenticationChainPath`|String|Path to a file containing a JSON list of Authenticator objects|null|yes|
```

Comment:
Hm, I think I'll change this to accept a list of JSON Strings, then rely on injectable config classes for per-Authenticator config. I was originally planning on having the file contain a list of "Authenticator specs".